### PR TITLE
Godot 4 Support

### PR DESCRIPTION
**Pull request by floriandotorg**
Hey @viniciusgerevini, thanks for your awesome work! I took the liberty to migrate it to Godot 4. I didn't test everything, but the main functionality is there.

### CODE REVIEW

**Comment by aaaaaaaaargh** [comment]
Haha, that's funny, I was about to address the same issue here :)

Looks like your work is very similar to what I've just done, so I'm commenting on that if you don't mind.

File: addons/AsepriteWizard/animated_sprite/import_plugin.gd [hunk 1]
```diff
@@ -58,7 +63,7 @@ func get_import_options(i):
 	]
 
 
-func get_option_visibility(option, options):
+func _get_option_visibility(option, options):
```
Comment (aaaaaaaaargh) [inline addons/AsepriteWizard/animated_sprite/import_plugin.gd:66]: This is missing the `path` parameter -> `func _get_option_visibility(path, option, options):`

Comment (floriandotorg) [reply addons/AsepriteWizard/animated_sprite/import_plugin.gd]: Thanks @aaaaaaaaargh! I changed that. Your comments are highly welcome. One thing that is still not working are the error messages, if you have a fix here, I'd love to integrate it.

**Comment by viniciusgerevini** [comment]
Thanks @floriandotorg . That's great. As this is not backwards compatible, and Godot 4 is still in alpha, would you mind opening this PR against a version 4 branch. I will keep the main branch compatible with version 3 until 4 becomes stable.

Overall it's looking great. I need to download version 4 to helping you test it, though. :)

**Comment by viniciusgerevini** [comment]
I created a `godot4` branch. I don't think you are able to change the base branch in the PR, so you might have to open a new one.

**Comment by floriandotorg** [comment]
Hey @viniciusgerevini, I'm so sorry, I overlooked your comment, just changed the base branch, hope it's still helpful :-)

File: addons/AsepriteWizard/animated_sprite/import_plugin.gd (hunk 1 above)
Comment (mvonkeil) [reply addons/AsepriteWizard/animated_sprite/import_plugin.gd]: @floriandotorg Sorry, I was overlooking your comment as well -_- 

I did have a patch but I must have forgotten to create a PR and it seems to be lost now. Sorry about that, but if you need any further assistance here I'd be happy to help.

**Pull request merged**